tests/src/Common/Platform/platformdefines.h

```diff
@@ -55,6 +55,11 @@ typedef const WCHAR *LPCWSTR, *PCWSTR;
 #define __stdcall
 #define _cdecl
 #endif
+#ifndef CROSS_MSC_VER_DEFINES
+#define CROSS_MSC_VER_DEFINES
+#define __FUNCSIG__ "<clang-3.5 signature not supported>"
+#define __int64 long long
+#endif // CROSS_MSC_VER_DEFINES
```

Comment:
I see. Because I had seen the build system the first time I was been able to know about the intricacies of it. I will investigate more carefully and fix that.